### PR TITLE
wp-admin Site Default: Add loading notice while changing the admin interface style

### PR DESCRIPTION
**Pull request by sejas**
- Related to https://github.com/Automattic/wp-calypso/pull/83297

## Proposed Changes

* Let's add a loading notice when the `Admin interface style` is changing to improve the UX

## Testing Instructions

* Navigate to Hosting Configuration on an atomic site
* Find the `Admin interface style` card
* Click on the unchecked radio button
* Verify that the radio buttons work as expected, and there is a loading notice

## Screencast

https://github.com/Automattic/wp-calypso/assets/779993/66c1f75d-b378-4427-81ff-0e9d3670c0cd

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by katinthehatsite — COMMENTED** [review]
The changes with the notice look good to me and it displays as expected while the changing of the interface is in progress:

Screenshot 2023-12-14 at 7 01 57 AM

File: client/components/command-palette/use-command-palette.tsx [hunk 1]
```diff
@@ -82,7 +82,9 @@ export const useCommandPalette = ( {
 
 	// Sort sites in the nested commands to be consistent with site switcher and /sites page
 	const { sitesSorting } = useSitesSorting();
-	const sortedSites = useSitesListSorting( allSites, sitesSorting );
+	const sortedSites = useSitesListSorting( allSites, sitesSorting ).filter(
```
Comment (katinthehatsite) [inline client/components/command-palette/use-command-palette.tsx]: Are these changes related to this PR and why are we adding them?

**Comment by katinthehatsite** [comment]
I left one comment but other than that, the changes look good to me!

File: client/components/command-palette/use-command-palette.tsx (hunk 1 above)
Comment (sejas) [reply client/components/command-palette/use-command-palette.tsx]: Removed! it was debugging code. Thanks for catching it!  https://github.com/Automattic/wp-calypso/pull/85262/commits/77c895c2f0f4c0cc6f34c84f6ce0f2b338413e3c

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/10289868

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @sejas, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Changing admin interface style…`

Thank you in advance!

**Comment by danielbachhuber** [comment]
Looks better, nice!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/10289868).
